docs/src/examples/components/Dropdown/Variations/DropdownExampleMultipleSearchImageAndContent.shorthand.tsx

```diff
@@ -55,7 +55,6 @@ const DropdownExample = () => (
     getA11yStatusMessage={getA11yStatusMessage}
     search
     getA11ySelectionMessage={getA11ySelectionMessage}
-    noResultsMessage="We couldn't find any matches."
```

Comment:
not completely sure that there should be any default value, given that this value needs to be properly localized. The reasoning behind this thought is the following: suppose that I, as a client, am using this component. I am not expecting any English text to show up in my French UI. 

Then if I wouldn't carefully study the API and spot all the places where default values may be provided, I will be trapped by this "alien" locale chiming into my UI. In that case I would rather prefer explicitness in a sense that there won't be any string that I am unaware of being shown to the client.

Reply:
Reverted this change, we will not provide a default message